### ISSUE TITLE
Pug Block Expansion feature causes unexpected vue/valid-v-slot

### ISSUE THREAD

**Issue by Grawl** [rashfael/eslint-plugin-vue-pug#9]
I use Pug [Block Expansion](https://pugjs.org/language/tags.html#block-expansion) feature to control elements nesting. But this causes errors sometimes.

Unexpected error:

```pug
MyComponent
	template(v-slot:foo)
		.foo: button(
			@click='handler'
		)
	template(v-slot:bar)
		span
```

`template(v-slot:bar)` reports:

```
ESLint: 'v-slot' directive must be owned by a custom element, but 'template' is not.(vue/valid-v-slot)
```

No error if do not chop down `q-btn` attributes:

```pug
MyComponent
	template(v-slot:foo)
		.foo: button(@click='handler')
	template(v-slot:bar)
		span
```

No error if I do not use Block Expansion:

```pug
MyComponent
	template(v-slot:foo)
		.foo
			button(
				@click='handler'
			)
	template(v-slot:bar)
		span
```
